x-pack/plugins/security_solution/cypress/e2e/detection_response/rule_management/rule_actions/bulk_actions/bulk_edit_rules_actions.cy.ts

```diff
@@ -68,154 +70,158 @@ const expectedExistingSlackMessage = 'Existing slack action';
 const expectedSlackMessage = 'Slack action test message';
 
 // TODO: Fix flakiness and unskip https://github.com/elastic/kibana/issues/154721
-describe.skip('Detection rules, bulk edit of rule actions', () => {
-  before(() => {
-    cleanKibana();
-    login();
-  });
-
-  beforeEach(() => {
-    deleteAlertsAndRules();
-    deleteConnectors();
-    cy.task('esArchiverResetKibana');
-
-    createSlackConnector().then(({ body }) => {
-      const actions: RuleActionArray = [
-        {
-          id: body.id,
-          action_type_id: '.slack',
-          group: 'default',
-          params: {
-            message: expectedExistingSlackMessage,
-          },
-          frequency: {
-            summary: true,
-            throttle: null,
-            notifyWhen: 'onActiveAlert',
-          },
-        },
-      ];
-
-      createRule(getNewRule({ name: ruleNameToAssert, rule_id: '1', max_signals: 500, actions }));
+describe.skip(
+  'Detection rules, bulk edit of rule actions',
+  { tags: [tag.ESS, tag.SERVERLESS] },
```

Comment:
So this was skipped due to flakiness, let's not tag it as `SERVERLESS` for now

Reply:
We should figure out a good way to track which tests are not tagged for serverless because of flake vs because the functionality is not available in serverless.

Reply:
@yctercero you can create any tag you believe might be helpful :)

Reply:
++ I think we could either use the already added tag `BROKEN_IN_SERVERLESS`, or add a new one `FLAKY_IN_SERVERLESS`